extension-manifest-v3/src/utils/trackerdb.js

```diff
@@ -62,3 +91,14 @@ export async function getPattern(key) {
 
   return patterns.get(key);
 }
+
+export function isCategoryBlockedByDefault(categoryId) {
+  switch (categoryId) {
```

Comment:
the blocking categories are:
* advertising
* porn_advertising
* email
* site_analytics

Reply:
This condition is the opposite - by default all categories are blocked except from listed here. Is there missing there something? Essential and CDN? Something more?

Reply:
yes, quite a few
https://github.com/ghostery/trackerdb/tree/main/db/categories

Not sure what's better, but I think it is more likely we add a new non-blocking category.